streams-designer/README.md

```diff
@@ -45,15 +45,31 @@ These are the IBM Bluemix services required to run this tutorial:
 Cloud Object Storage instance was not created beforehand, you will have the option of adding one.
 ![Create New Project in Watson Studio](create-new-project-watson-studio.png) 
 
-## Create a Streams flow
+## Option 1: Import the Streams flow file
+1. Create a new Streams flow under the `Add to project` drop-down list.
+![Select Streams flow](streams-flow-watson-studio.png)
+1. On the **New Streams Flow** screen, Select **From file**.  Drag and drop [Cloudant_to_Db2_streams_flow.stp](Cloudant_to_Db2_streams_flow.stp)
```

Comment:
Doesn't this file need to be uploaded too?

Reply:
Yes that would be helpful 😄 .  Added in e19db09.